middleware/util/logging/configuration.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"encoding/json"
+
+	"github.com/digitalbitbox/bitbox-wallet-app/util/errp"
+	"github.com/sirupsen/logrus"
+)
+
+// Configuration serializes and deserializes the logging parameters.
+type Configuration struct {
+	// Output location of the logger.
+	// Can be either a path relative to the configuration directory, STDOUT or STDERR.
+	Output string `json:"output"`
+
+	// Level from which on the entries are logged.
+	Level logrus.Level `json:"level"`
+}
+
+// MarshalJSON implements json.Marshaler.
+func (configuration Configuration) MarshalJSON() ([]byte, error) {
+	return json.Marshal(&map[string]string{
+		"output": configuration.Output,
+		"level":  configuration.Level.String(),
+	})
+}
+
+// UnmarshalJSON implements json.Unmarshaler.
```

Comment:
Nit: Typically Go comments are on the form of `Foo returns Bar`. I.e here, maybe:
```
UnmarshalJSON returns non-nil error if `bytes` can't be unmarshalled.

UnmarshalJSON implements the json.Unmarshaler interface.
```

(Or we could move the `implements the json.Unmarshaler interface.` part up to the type definition.)